app/src/main/java/co/pxhouse/sas/arch/model/_Providers.kt

```diff
@@ -74,10 +74,16 @@ fun generateProviders(context: Context, persistedValues: PersistedValues) = list
         "https://www.qwant.com/?q=%s",
         5
     ),
+    StaticSearchProvider(
+        R.drawable.ic_provider_startpage,
+        "Startpage",
+        "https://www.startpage.com/do/search?query=%s",
+        6
+    ),
     CustomSearchProvider(
         R.drawable.ic_provider_generic,
         context.getString(R.string.custom_provider),
         persistedValues,
-        6
+        7
```

Comment:
This is an ID that's persisted. 
Please keep 6 for CustomSearchProvider and use 7 for startpage. The order in the list defines the order on the UI.